gammapy/spectrum/tests/test_observation.py

```diff
@@ -90,6 +90,60 @@ def test_spectrum_observation_3():
     tester = SpectrumObservationTester(obs, pars)
     tester.test_all()
 
+@requires_dependency('scipy')
+@requires_dependency('sherpa')
+@requires_data('gammapy-extra')
+def make_observation_list():
+    """obs without IRF"""
+    nbin=10
+    energy = np.logspace(-1, 1, nbin+1) * u.TeV
+    livetime = 2 * u.h
+    data=np.arange(nbin)
+    data1=data
+    data2=data
+    data3=data
```

Comment:
This won't work since ``data1``, ``data2``, and ``data3`` all point to the same object. This is because in Python what you are used to calling varibles, are not really varibales (i.e. memory locations) but just names for certain objects. This is also explained [here](https://stackoverflow.com/questions/20699736/why-should-i-refer-to-names-and-binding-in-python-instead-of-variables-and).

Let me give you an example
```
>>> val = np.arange(10)
>>> a = val
>>> b = val
>>> a is b
True
>>> a[0] = -5
>>> b
array([-5,  1,  2,  3,  4,  5,  6,  7,  8,  9])
```

If you want to create two different objects you can do
```
>>> a = np.arange(10)
>>> b = a.copy()
>>> a is b
False
```